pywatts/summaries/tsne_visualisation.py

```diff
@@ -0,0 +1,134 @@
+from typing import Dict
+
+import matplotlib.pyplot as plt
+import tikzplotlib
+import xarray as xr
+from pywatts.core.base_summary import BaseSummary
+from sklearn.manifold import TSNE
+import numpy as np
+
+from pywatts.core.summary_object import SummaryObjectList
+
+
+class TSNESummary(BaseSummary):
+    """
+    Summary that performs a TSNE to visualise the data. It is possible to specify mask by providing kwargs
```

Comment:
I would once write it out and would use t-SNE instead of TSNE:
Summary that performs a t-distributed stochastic neighbor embedding (t-SNE) to visualise the data. It is possible to specify masks by providing kwargs